charts/logan/values.yaml

```diff
@@ -424,6 +434,125 @@ fluentd:
+            #ociLALogGroupID:
+            # Worker number in case of multi process workers enabled. If not set when multi process workers enabled, then it defaults to 0.
+            #worker:
+         audit:
+            sqsQueue: "audit"
+            # S3 object key
+            objectKey: .*?kube-apiserver-audit
+            # Logging Analytics log source to use for parsing and processing EKS Control Plane Audit Logs.
+            ociLALogSourceName: "Kubernetes Audit Logs"
+            #metadata:
+               #"Client Host Region": "America"
+               #"Environment": "Production"
+               #"Third Key": "Third Value"
+            #ociLAEntityID:
+            #ociLALogGroupID:
+            #worker:
+         authenticator:
+            cwLogStreamName: "authenticator"
+            sqsQueue: "authenticator"
+            # S3 object key
+            objectKey: .*?authenticator
+            # Logging Analytics log source to use for parsing and processing EKS Control Plane Authenticator Logs.
+            ociLALogSourceName: "AWS EKS Authenticator Logs"
+            #metadata:
+               #"Client Host Region": "America"
+               #"Environment": "Production"
+               #"Third Key": "Third Value"
+            #ociLAEntityID:
+            #ociLALogGroupID:
+            #worker:
+         kubecontrollermanager:
+            cwLogStreamName: "kube-controller-manager"
+            sqsQueue: "kube-controller-manager"
+            # S3 object key
+            objectKey: .*?kube-controller-manager
+            # Logging Analytics log source to use for parsing and processing EKS Control Plane Kube Controller Manager Logs.
+            ociLALogSourceName: "Kubernetes Controller Manager Logs"
+            #metadata:
+               #"Client Host Region": "America"
+               #"Environment": "Production"
+               #"Third Key": "Third Value"
+            #ociLAEntityID:
+            #ociLALogGroupID:
+            #worker:
+         cloudcontrollermanager:
+            cwLogStreamName: "cloud-controller-manager"
+            sqsQueue: "cloud-controller-manager"
+            # S3 object key
+            objectKey: .*?cloud-controller-manager
+            # Logging Analytics log source to use for parsing and processing EKS Control Plane Cloud Controller Manager Logs.
+            ociLALogSourceName: "Cloud Controller Manager Logs"
+            #metadata:
+               #"Client Host Region": "America"
+               #"Environment": "Production"
+               #"Third Key": "Third Value"
+            #ociLAEntityID:
+            #ociLALogGroupID:
+            #worker:
+         scheduler:
+            cwLogStreamName: "kube-scheduler"
+            sqsQueue: "scheduler"
+            # S3 object key
+            objectKey: .*?kube-scheduler
+            # Logging Analytics log source to use for parsing and processing EKS Control Plane Scheduler Logs.
+            ociLALogSourceName: "Kubernetes Scheduler Logs"
```

Comment:
most of these logs using Kubernetes Standard Log Format parser which is a multi-line. Should we add the multi-line regex ?